src/evaluator/index.ts

```diff
@@ -15,6 +15,14 @@ const treatmentException = {
   config: null
 };
 
+function treatmentsException(splitNames: string[]) {
```

Comment:
we use TS in strict mode, but we try to avoid using types when they are automatically inferred by TS. A case where we would need to define the return type of a function is when the inferred TS type is not as specific as we want.